beta/kustomize/overlays/workspace/workspace-settings.json

```
@@ -0,0 +1,4 @@
+{
+  "actor": "admin.email.goes.here@example.com",
+  "bridgeAddress": "https://bridge.example.com"
```

Same nitpick as above:

```suggestion
  "bridgeAddress": "https://scim.example.com"
```